NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -91,132 +182,135 @@ def run(
             Lower and upper bound of the frequency range in which noise shall be
             added
         """
+
+        # check that or all channels channel.get_frequencies() is identical
+        last_freqs = None
+        for channel in station.iter_channels():
+            if (not last_freqs is None) and (
+                    not np.allclose(last_freqs, channel.get_frequencies(), rtol=0, atol=0.1 * units.MHz)):
+                logger.error("The frequencies of each channel must be the same, but they are not!")
+                return
+
+            last_freqs = channel.get_frequencies()
```

Comment:
I think these were also changes that Moritz implemented - I am also not sure in which case they would be needed, but he clearly deemed this important. I also think we can keep it since it does not seem to cause any problems

Reply:
But only returning an error and than not adding the noise? Wouldn't it be better to raise an error?
This might make it a bit more readable? 
```
        # check that or all channels channel.get_frequencies() is identical
        freqs = station.get_channel(station.get_channel_ids()[0]).get_frequencies()
        for channel in station.iter_channels():
            if not np.allclose(freqs, channel.get_frequencies(), rtol=0, atol=0.1 * units.MHz):
                logger.error("The frequencies of each channel must be the same, but they are not!")
```